app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -116,7 +116,7 @@ function buildArticleHTML(article) {
     return '<div class="single-article single-article-small-pic" data-content-user-id="'+article.user_id+'">\
       '+videoHTML+'\
       '+orgHeadline+'\
-      <div class="small-pic">\
+      <div class="small-pic" data-hover-details="users/'+article.user_id+'">\
```

Comment:
shouldn't this be the API path? If you replace it with `api_user_path(article.user_id)` you don't need to construct the API path later on

Reply:
I was thinking that this establishes a naming convention for `data-hover-details` which can apply to other stuff like `tags/1` etc. but it doesn't necessarily mean we definitely want to define the functionality that _uses_ this in the future.

`users/1` tells us it's user with ID 1 we're looking for, but if we want to change how we get that later, we're not putting too much right here.

My logic about putting this in now is that we're going to have to either clear a bunch of production caches or wait for them to clear before this becomes mostly widely available. So rather than forcing this issue I thought maybe we deploy this now and do the rest later. 🤷‍♂️

Reply:
> I was thinking that this establishes a naming convention for data-hover-details which can apply to other stuff like tags/1 etc. but it doesn't necessarily mean we definitely want to define the functionality that uses this in the future.

I mentioned this because it looked like you were going to build paths to use for an eventual API call as you mentioned in the description, so it made sense to just put the full path and just `GET` that in the JS, it's also more robust than to build paths in JS (and it will tell us at a glance which resource it refers to) because it'll be `data-hover-details="/api/users/123"` already

I'm not sure I understood the connection with the edge caching keys and the partial but if you're trying to use the record cache key you can also just replace that with `record.cache_key`